modules/apps/remote-app/remote-app-api/src/main/java/com/liferay/remote/app/model/RemoteAppEntrySoap.java

```diff
@@ -233,6 +234,18 @@ public void setCustomElementURLs(String customElementURLs) {
 		_customElementURLs = customElementURLs;
 	}
 
+	public boolean getCustomElementUseESM() {
```

Comment:
Why both getCustomElementUseESM and isCustomElementUseESM?

I will leave just the isCustomElementUseESM as I think it is the correct name pattern for boolean properties